internal/stack/variants.go

```diff
@@ -6,20 +6,34 @@ package stack
 
 import (
 	"fmt"
-	"strings"
+
+	"github.com/Masterminds/semver/v3"
```

Comment:
just a small comment, I see you imported the "v3". Is there anything missing in the `github.com/Masterminds/semver` as we are using the base release? If not, would you mind switching to that version?

Reply:
Done.
I had some trouble finding the correct v1 syntax for the range constraint as it differs from the v3 a bit.